app/_locales/en/messages.json

```diff
@@ -1019,10 +1019,10 @@
     "message": "I have acknowledged the alert and still want to proceed"
   },
   "confirmAlertModalDetails": {
```

Comment:
To change the copy correctly and trigger new translations, you can add the new copy under a new translation key, and delete the old key in all languages.

Reply:
Oh, thanks for pointing that out. I've applied the recommended changes. https://github.com/MetaMask/metamask-extension/pull/27205/commits/474affa1d00b55176e5c379fde23fa1f9a5b3f55